views/random-beer.hbs

```diff
@@ -0,0 +1,13 @@
+
+{{#each randomBeer}}
+  {{> beerpartial this}}
+  <ul>
+    {{#each this.food_pairing}}
+        <li>{{this}}</li>
+    {{/each}}
+  </ul>
+  <p>
+    {{brewers_tips}}
+  </p>
+{{/each}}
+
```

Comment:
A small thing, but it would be nice to introduce what that text is: <p> Food pairing: </p> and then <ul> etc.